control_plane/storcon_cli/src/main.rs

```diff
@@ -0,0 +1,643 @@
+                operations: Vec<Operation>,
+            }
+
+            let req = PageserverTenantMigrateRequest {
+                tenant_id: tenant_id.to_string(),
+                target_pageserver_id,
+                skip_check_availability: true,
+            };
+
+            let response = storcon_client
+                .dispatch::<PageserverTenantMigrateRequest, OperationResponse>(
+                    Method::POST,
+                    "management/api/v2/pageservers/migrate_tenant".to_string(),
+                    Some(req),
+                )
+                .await?;
+            println!("{}", serde_json::to_string(&response).unwrap());
+        }
+        Command::TenantShardSplit {
+            tenant_id,
+            shard_count,
+            stripe_size,
+        } => {
+            let req = TenantShardSplitRequest {
+                new_shard_count: shard_count,
+                new_stripe_size: stripe_size.map(ShardStripeSize),
+            };
+
+            let response = storcon_client
+                .dispatch::<TenantShardSplitRequest, TenantShardSplitResponse>(
+                    Method::PUT,
+                    format!("control/v1/tenant/{tenant_id}/shard_split"),
+                    Some(req),
+                )
+                .await?;
+            println!(
+                "Split tenant {} into {} shards: {}",
+                tenant_id,
+                shard_count,
+                response
+                    .new_shards
+                    .iter()
+                    .map(|s| format!("{:?}", s))
+                    .collect::<Vec<_>>()
+                    .join(",")
+            );
+        }
+        Command::TenantShardMigrate {
+            tenant_shard_id,
+            node,
+        } => {
+            let req = TenantShardMigrateRequest {
+                tenant_shard_id,
+                node_id: node,
+            };
+
+            storcon_client
+                .dispatch::<TenantShardMigrateRequest, TenantShardMigrateResponse>(
+                    Method::PUT,
+                    format!("control/v1/tenant/{tenant_shard_id}/migrate"),
+                    Some(req),
+                )
+                .await?;
+        }
+        Command::TenantConfig { tenant_id, config } => {
+            let tenant_conf = serde_json::from_str(&config)?;
+
+            vps_client
+                .location_config(
+                    TenantShardId::unsharded(tenant_id),
+                    LocationConfig {
+                        mode: pageserver_api::models::LocationConfigMode::AttachedSingle,
+                        generation: None,
+                        secondary_conf: None,
+                        shard_number: 0,
+                        shard_count: 0,
+                        shard_stripe_size: 0,
+                        tenant_conf,
+                    },
+                    None,
+                    false,
+                )
+                .await?;
+        }
```

Comment:
Why not call it `Command::LocationConfig` ? We want people to start learning the concept of locations, don't we?

Reply:
This command was specifically for setting TenantConfig, but written before I had a specific API for that (so used location_config) -- I've updated it to use a narrower API so that the name makes more sense in 61232be82b27bbc0c699779c8dbf40ee61132ed6